kingpin/actors/aws/elb.py

```diff
@@ -362,7 +362,93 @@ def _execute(self):
             self.log.info('Would instruct %s to use %s' % (
                 self.option('name'), self.option('cert_name')))
         else:
-            yield self._use_cert(elb, cert_arn)
+            yield self._use_cert(elb, cert_arn, self.option('port'))
+
+
+class UpdateCert(ELBBaseActor):
```

Comment:
I'm thinking that maybe this name should change -- `UpdateCertReferences`? I just find it a little vague as it is now. Thoughts?